nebula/files/nebula.init

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh /etc/rc.common
+# Copyright 2021 Stan Grishin (stangri@melmac.net)
+# shellcheck disable=SC2039
+PKG_VERSION='dev-test'
```

Comment:
I think this is a leftover from testing and I am not sure if you want to have it here, because this will just add maintenance overhead and it is possible that you will forget to update it sometimes.

Reply:
This gets replaced automatically in the Makefile like in my other packages.

Reply:
Ok.